FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -139,6 +166,17 @@ NS_SWIFT_NAME(RemoteConfigSettings)
 @property(nonatomic, assign) NSTimeInterval fetchTimeout;
 @end
 
+#pragma mark - FIRRemoteConfigUpdate
+/// Firebase Remote Config update
+NS_SWIFT_NAME(RemoteConfigUpdate)
+@interface FIRRemoteConfigUpdate : NSObject
+
+/// Parameter keys whose values have been updated from the currently activated values. Includes
+/// keys that are added, deleted, and whose value, value source, or metadata has changed.
+@property(nonatomic, readonly, nonnull) NSSet<NSString *> *updatedKeys;
```

Comment:
this doesn't seem to be in the API review.

Reply:
It's apart of an addendum: https://docs.google.com/document/d/1q8ui1i7DXXjIM7pu9d_LeJIoSwNrZd_nimMzJCwdruA/edit#heading=h.x94y9zzc13xn